website/profile/utils.py

```diff
@@ -86,10 +87,14 @@ def serialize_user(user, node=None, admin=False, full=False, is_profile=False, i
         else:
             merged_by = None
 
+        default_storage_region = user.get_addon('osfstorage').default_region
+        region_list = [region for region in Region.objects.all().values('_id', 'name')]
```

Comment:
Could you use your `get_storage_region_list` function here?

Reply:
We could, but it wouldn't actually decrease the amount of code because we need the `default_storage_region` defined explicitly here, not just have the it be the first thing on the list.

Reply:
Hmm, could you pass the whole list and just pop the first [here](https://github.com/CenterForOpenScience/osf.io/pull/8386/files#diff-6b92464a8f46e4590fe59579570e096eR155)? Not a big deal either way though.